homeassistant/components/rfxtrx/sensor.py

```diff
@@ -204,6 +204,9 @@ class RfxtrxSensorEntityDescription(SensorEntityDescription):
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=UV_INDEX,
     ),
+    RfxtrxSensorEntityDescription(
```

Comment:
Some unrelated changes seem to have been included.

Reply:
😳 thx